zips/zip-0246.rst

```diff
@@ -379,8 +375,6 @@ In case the transaction has no memo chunks, ``memo_digest`` is::
 
     BLAKE2b-256("ZTxIdMemo___Hash", [])
 
-The field encodings are specified in ZIP 230 [#zip-0230-issuance-field-encodings]_.
```

Comment:
This shouldn't have been removed; instead its URL link should have been fixed to point to the part of ZIP 230 containing the spec for the fields used in `memo_digest`.